source/includes/_bonus_programs.md.erb

```diff
@@ -0,0 +1,357 @@
+
+`DELETE <%= config[:host] %>/bonus_cards/#{id}`
+
+Параметры запроса:
+
+Параметр|Обязательный|Описание
+---|---
+id|Да|Идентификатор бонусной карты
+
+## API для получения карт и типов карт, которые можно применить к заказу
+
+```shell
+curl "<%= config[:host] %>/bonus_cards?order_number=R123456" \
+  -H "Authorization: Token token=#{TOKEN}"
+```
+> Ответ будет содержать следующие данные:
+
+```json
+  {
+    "bonus_cards": [{
+      "id": 121,
+      "number": "1234567890",
+      "bonus_PROGRam_id": 1
+    }]
+  }
+```
+
+Получить список бонусных карт, которые могут быть применены к заказу можно, выполнив запрос:
+
+`GET <%= config[:host] %>/bonus_cards?order_number=#{order_number}`
+
+Параметры запроса:
+
+Параметр|Обязательный|Описание
+---|---
+order_number|Да|Номер заказа
+
+Параметры ответа:
+
+Параметр|Описание
+---|---
+id|Идентификатор бонусной карты
+number|Номер бонусной карты
+bonus_program_id|Идентификатор бонусной программы
+
+### Список бонусных программ, которые могу быть применены к данному заказу:
+
+```shell
+curl "<%= config[:host] %>/bonus_programs?order_number=R1234567" \
+  -H "Authorization: Token token=#{TOKEN}"
+```
+
+> Ответ будет содержать следующие данные:
+
+```json
+  {
+    "bonus_programs": [{
+      "id": 1,
+      "name": "«Аэрофлот Бонус»",
+      "icon": {
+        "url": "https://<%= config[:host] %>/public/loyalty_cards_logos/16/mini/aero-img.png"
+      }
+    }]
+  }
+```
+
+Получить список бонусных программ, которые могут быть применены к заказу, можно, выполнив запрос:
+
+`GET <%= config[:host] %>/bonus_programs?order_number=#{order_number}`
+
+Параметры запроса:
+
+Параметр|Обязательный|Описание
+---|---
+order_number|Да|Номер заказа
+
+Вернется массив бонусных программ следующей структуры:
+
+Параметр|Описание
+---|---
+id|Идентификатор бонусной программы
+name|Наименование бонусной программы
+icon|Логотип бонусной программы
+
+## API применения карт к заказу
+
+При совершении заказа, пользователь может выбрать одну из бонусных программ для начисления бонусов при покупке.
+
+### Получение данных по бонусной карте, которая будет применена к заказу:
+
+```shell
+curl "<%= config[:host] %>/orders/R010203/bonus_card_action" \
+  -H "Authorization: Token token=#{TOKEN}"
+```
+> Ответ будет содержать следующие данные:
+
+```json
+  {
+    "points": "201 миля",
+    "bonus_card": {
+      "id": 121,
+      "number": "1234567890",
+      "bonus_program_id": 1
+    }
+  }
+```
+
+Получить выбранную бонусную карту пользователя можно, выполнив запрос:
+
+`GET <%= config[:host] %>/orders/#{number}/bonus_card_action`
+
+Параметры запроса:
+
+Параметр|Обязательный|Описание
+---|---
+number|Да|Номер заказа
+
+Параметры ответа:
+
+Параметр|Описание
+---|---
+points|Количество начисляемых баллов по карте
+bonus_card|Описание бонусной карты
+
+Структура объекта bonus_card:
+
+Параметр|Описание
+---|---
+id|Идентификатор бонусной карты
+number|Номер бонусной карты
+bonus_program_id|Идентификатор бонусной программы
```

Comment:
Get rid of duplicated response descriptions.

Reply:
fixed